vunit/modelsim_interface.py

```diff
@@ -276,7 +276,8 @@ def _create_load_function(self,  # pylint: disable=too-many-arguments,too-many-l
         else:
             coverage_file = join(output_path, "coverage.ucdb")
             self._coverage_files.add(coverage_file)
-            coverage_save_cmd = "coverage save -onexit -assert -directive -cvg -codeAll {%s}" % fix_path(coverage_file)
+            coverage_save_cmd = "coverage save -onexit -assert -directive -cvg -codeAll {%s}" \
+			% (os.path.basename(os.path.dirname(output_path)), fix_path(coverage_file))
```

Comment:
In vunit the outpath always contains the testname. Hereafter the attributes of the generated coverage report with the testname:
`
# coverage read -dataset coverage vunit_out_rtl/tests/tb_lib.tb_mybloc.rtl.testcase_bfm/modelsim/coverage.ucdb
# vunit_out_rtl/tests/tb_lib.tb_mybloc.rtl.testcase_bfm/modelsim/coverage.ucdb opened as coverage dataset "coverage"
coverage attribute
coverage attribute
# TESTNAME = tb_lib.tb_mybloc.rtl.testcase_bfm
#     SIMTIME             = 1182250.000000
#     TIMEUNIT            = ps
`